cosmovisor/logger.go

```diff
@@ -0,0 +1,15 @@
+package cosmovisor
+
+import (
+	"os"
+	"time"
+
+	"github.com/rs/zerolog"
+)
+
+var Logger zerolog.Logger
+
+func SetupLogging() {
```

Comment:
Not meaning to block here, but is there a reason to use a global here instead of just returning a logger instance?

Reply:
You mean to have a function which returns a private instance? Or putting it in a context?

I think this is fine for the moment - it's still better than using `fmt.Print*`. Logger is not something that it's going to change during the runtime.

Reply:
@spoo-bar  - since nobody is blocking, feel free to add `automerge` unless you want to expand in this thread.

Reply:
Im not sure what you mean by private, but I mean just _a_ instance, instead of relying on a global. So instead of requiring a `SetupLogger` call, you just call `logger := CreateLogger()`.

Reply:
@alexanderbez  creating an instance would require `logger` to be passed around to each function.
having a global makes it easy to call it from anywhere.

Reply:
> creating an instance would require logger to be passed around to each function.

Not really, as the caller would would just call the function when it needs a logger (this is what SDK modules do BTW).

Also, globals are generally the root of all evil. Avoid them if possible (in most contexts).

Reply:
Ah okay. 
But it would still require frequent reinitialization all over. Idk if having it has a global for a small component like cosmovisor is all that bad, is it?